packages/cardano-services/src/ChainHistory/DbSyncChainHistory/queries.ts

```diff
@@ -173,13 +173,40 @@ export const findRedeemersByTxIds = `
 	WHERE tx.id = ANY($1)
 	ORDER BY redeemer.id ASC`;
 
+export const findVotingProceduresByTxIds = `
+	SELECT
+		tx.hash AS tx_id,
+		voter_role,
+		committee_voter,
+		dh.raw AS drep_voter,
+		dh.has_script AS drep_has_script,
+		ph.hash_raW as pool_voter,
```

Comment:
We might need the raw data at some point, but for display purposes we probably just want to bech32
```suggestion
		ph.view as pool_voter,
```
if not we'll still need to address the typo `hash_raW` -> `hash_raw`